portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/FileInstall.testcase

```diff
@@ -135,7 +135,9 @@ definition {
 
 		GogoShell.openGogoShellAdmin();
 
-		AssertConsoleTextNotPresent(value1 = "STARTED com.liferay.portal.cluster.multiple.sample_");
+		// Ignore test until LPS-146449 is resolved
+
+		/* AssertConsoleTextNotPresent(value1 = "STARTED com.liferay.portal.cluster.multiple.sample_");*/
```

Comment:
Use Single line comment for both lines, this is using a "multiline" comment for the second line

Change to "Ignore assert until ..." We are not ignoring the whole test.